Assets/Scripts/UIAndMenus/SettingsController.cs

```diff
@@ -35,9 +35,9 @@ public class SettingsController : MonoBehaviour
     // Start is called before the first frame update
```

Comment:
This class is starting to get pretty long, so at some point it should probably be refactored into multiple classes (maybe separating the categories logic into a different class and/or putting the logic for specific features--like adjusting the resolution and changing the volume--into their own classes).

Reply:
Agreed, I'm going to push this off a bit and make a new work item